### DISCUSSION TITLE
Zoo Attack. Hi I am trying to generate 1000 of zoo adversarial samples using cifar10. Estimatatedly it will take 1000 hours to generate it in a server with GPU. I wonder if anyone has an idea on how to do it quicker. Thanks

### DISCUSSION THREAD

**Discussion by izakariyya** [Trusted-AI/adversarial-robustness-toolbox#2098] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**System information (please complete the following information):**
 - OS
 - Python version
 - ART version or commit number
 - TensorFlow / Keras / PyTorch / MXNet version

**Comment by beat-buesser** [comment]
Hi @izakariyya I convert this issue into a discussion to continue it under the Discussion tab above.

**Reply by beat-buesser** [reply]
Have you checked if the GPU is being utilized?
